packages/e2e-tests/src/tests/SessionSigs.spec.ts

```diff
@@ -0,0 +1,825 @@
+    {
+      accessControlConditions: accs,
+      dataToEncrypt: 'Hello world',
+    },
+    devEnv.litNodeClient as unknown as ILitNodeClient
+  );
+
+  // -- Expected output:
+  // {
+  //   ciphertext: "pSP1Rq4xdyLBzSghZ3DtTtHp2UL7/z45U2JDOQho/WXjd2ntr4IS8BJfqJ7TC2U4CmktrvbVT3edoXJgFqsE7vy9uNrBUyUSTuUdHLfDVMIgh4a7fqMxsdQdkWZjHign3JOaVBihtOjAF5VthVena28D",
+  //   dataToEncryptHash: "64ec88ca00b268e5ba1a35678a1b5316d212f4f366b2477232534a8aeca37f3c",
+  // }
+
+  // -- assertions
+  expect(encryptRes.ciphertext).toBeDefined();
+
+  expect(encryptRes.dataToEncryptHash).toBeDefined();
+
+  const accsResourceString =
+    await LitAccessControlConditionResource.generateResourceString(
+      accs,
+      encryptRes.dataToEncryptHash
+    );
+
+  const litActionSessionSigs2 = await generator(devEnv, alice, [
+    {
+      resource: new LitAccessControlConditionResource(accsResourceString),
+      ability: LitAbility.AccessControlConditionDecryption,
+    },
+  ]);
+
+  // -- Decrypt the encrypted string
+  const decryptRes = await LitJsSdk.decryptToString(
+    {
+      accessControlConditions: accs,
+      ciphertext: encryptRes.ciphertext,
+      dataToEncryptHash: encryptRes.dataToEncryptHash,
+      sessionSigs: litActionSessionSigs2,
+      chain: 'ethereum',
+    },
+    devEnv.litNodeClient as unknown as ILitNodeClient
+  );
+
+  devEnv.releasePrivateKeyFromUser(alice);
+
+  expect(decryptRes).toEqual('Hello world');
+};
+
+const broadcastAndCollect = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+) => {
+  devEnv.setUnavailable(LIT_TESTNET.MANZANO);
+
+  const alice = await devEnv.createRandomPerson();
+  // set access control conditions for encrypting and decrypting
+  const accs = AccessControlConditions.getEmvBasicAccessControlConditions({
+    userAddress: alice.authMethodOwnedPkp?.ethAddress!,
+  });
+
+  const litActionSessionSigs = await generator(devEnv, alice);
+
+  const res = await devEnv.litNodeClient?.executeJs({
+    sessionSigs: litActionSessionSigs,
+    code: `(async () => {
+              let rand = Math.floor(Math.random() * 100);
+              const resp = await Lit.Actions.broadcastAndCollect({
+                  name: "temperature",
+                  value: rand.toString(),
+              });
+              Lit.Actions.setResponse({
+                  response: JSON.stringify(resp)
+              });
+            })();`,
+    jsParams: {},
+  });
+  devEnv.releasePrivateKeyFromUser(alice);
+
+  const response = res?.response;
+  expect(response).toBeDefined();
+};
+
+const decryptAndCombine = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+) => {
+  devEnv.setUnavailable(LIT_TESTNET.MANZANO);
```

Comment:
🔪 ideally this would be replaced with wrapping the test definition function call itself in a conditional check, see above comment.

That being said, do we _really_ never run any test that uses this function on manzano -- and should it be disabled on `datil-test` if it was disabled on `manzano`?